docs/storage-management.asciidoc

```diff
@@ -3,67 +3,159 @@
 
 [partintro]
 --
-In the following section you can learn how to:
 
-* <<manage-indices-kibana, Manage APM indices via Kibana>>
+* <<sizing-guide, Storage and sizing guide>>
+* <<processing-performance, Processing and performance>>
 * <<reduce-storage, Reduce storage usage>>
+* <<manage-indices-kibana, Manage APM indices via Kibana>>
 * <<update-existing-data, Update existing data>>
 --
 
-[[manage-indices-kibana]]
-== Manage Indices via Kibana
-The Kibana UI for managing indices allows you to
-view indices, index settings, mappings, docs count, used storage per index and much more.
-You can perform management operations,
-for example, deleting indices directly via the Kibana UI.
-The UI also supports applying bulk operations on several indices at once.
-Check out the {kibana-ref}/managing-indices.html[Kibana Managing Indices] documentation for more details.
+[[sizing-guide]]
+== Storage and sizing guide
+
+APM Server handles a few different kinds of {apm-overview-ref-v}/apm-data-model.html[events].
+Processing and storage costs are largely dominated by
+{apm-overview-ref-v}/transactions.html[transactions] and
```

Comment:
mention here or somewhere else that transactions dominate when sampling rate is very small (i think this distinction is important because is not very intuitive that sampling down still sends all transactions captured)